sagemaker-training-compiler/huggingface/pytorch_tune_batch_size/batch_tune.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        if 'result' in line and 'max_batch_size' in line or 'Total max batch' in line:
</pre><p>Check logic here - do you mean <code>if 'result' in line and ('max_batch_size' in line or 'Total max batch' in line)</code> ? Currently it will be effectively <code>if ('result' in line and 'max_batch_size' in line) or 'Total max batch' in line</code></p><p> </p><p>This also seems like a brittle way to print the logs since print statements could be changed. Any possible alternatives?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
This line captures two log statements that the training job script `find_max_batch_size.py` outputs.  The logic as written is correct as we are capturing two lines, one with "Total max batch" and one with the word "result".  Given that we control the training job script, the print statements will not change.